chapters/methodology/theory.tex

```diff
@@ -102,21 +121,24 @@ \subsection{The $\Omega$ Methods}
 the $\Omega$-methods will use a version of the adjoint scalar flux to
 consistently bias a Monte Carlo problem with the intention of reducing the
 variance. In section \ref{sec:Importance} the concept of importance was
-introduced. Notably, it was shown that the adjoint flux is a good marker for
+introduced. Notably, it was shown that the adjoint flux is a good marker 
 for the likelihood of particles to contribute to a tally, which is the
 particle's importance. It was also shown that the
 product of the forward and adjoint fluxes generates a pseudo-particle flux called
 the contributon flux, where contributons are ``importance particles''.
 These importance particles can be used to show preferential flow paths from a
 source to a tally or desired location.
+% Yes, this is the kind of material for this section. Good description!
 
 By using a version of the adjoint scalar flux that has been formulated with the
 contributon flux, the direction of particle flow will be incorporated into the
-importance map, and consequently also the
-variance reduction parameters. By using a variant of the adjoint scalar flux,
-the method will still maintain the importance relationship between particles and
+importance map and, consequently, the
+variance reduction parameters. By using this variant of the adjoint scalar flux,
+the method will maintain the importance relationship between particles and
 their proximity to the adjoint source location, or the forward response
-location. The adjusted adjoint scalar flux quantity, or the $\Omega$-adjoint
+location. 
+% I'm a little confused by the above sentence...
```

Comment:
Ah, okay. So in general we say that, importance increases as a particle gets closer to the adjoint source. In a standard adjoint calculation the importance is the same no matter where the particle is coming from. It just increases with importance with its proximity to the adjoint source. 

In CADIS we set adjoint source = forward response function, so I was trying to reiterate that as a particle gets closer to the adjoint source, which is also the forward response/tally, that its importance increases. We maintain that particular feature in our method, but we also incorporate the directionality of flow, so not *all* regions near the tally are important. However, even though directionality is incorporated into our method, the particle importance as it approaches the adjoint source/forward response will increase. 

Does that make sense?